src/app/config/routes.tsx

```diff
@@ -2,11 +2,15 @@ import React, { ComponentClass } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 import MainPage from '@core/pages/main';
+const MyDynamicRoute = React.lazy(() => import(/* webpackChunkName: "dynamic-route" */ '@core/pages/dynamic-route/dynamic-route'));
 
 export const AppRoutes = () => (
   <Router>
     <Switch>
       <Route path="/" exact component={MainPage as ComponentClass} />
+      <React.Suspense fallback={<div>Loading...</div>}>
```

Comment:
Do wee need Suspence here?  as a wrapper for route? I think rooute can be there and all the stuff should happend after enetring to it. So it looks like additional thing, but please check :)

Reply:
even if Suspense is needed, please import it properly, instead of using `React.Suspense` syntax

Reply:
fixed